data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1918 @@
+			label plan
+			`	He introduces you to the three Kimek, and goes to explain the plan. "Granted access to our civilian ships, you have been. Three new Kimek Spires, you have very recently bought. Many passengers to <destination>, in them you are carrying. A very lucrative job, it is." He pauses, and gives the Kimek permission to head out. "Pose as civilians, working in your ships, they will. Armed and ready to investigate <planet> thoroughly, the soldiers in those Spires are. Instructed to follow your ship there, these three were."`
+			`	Plortup wishes you all good luck, and you're escorted back to your ship. You see the three aforementioned Kimek Spires parked near the <ship>, and one of the Kimek Heliarchs hails you, saying they're ready to depart when you are.`
+				accept
+	npc accompany save
+		personality escort heroic opportunistic
+		government "Coalition"
+		ship "Kimek Spire" "Kiry Paichee"
+		ship "Kimek Spire" "Achipa Kyrri"
+		ship "Kimek Spire" "Pakee Pikary"
+	on visit
+		dialog `You've reached <planet>, but the Heliarch transports haven't arrived yet! Best depart and wait for them to get here.`
+
+mission "Heliarch Containment 4-B"
+	name "Undercover: Payment"
+	description "Collect your payment for escorting the Heliarch transports."
+	landing
+	to offer
+		has "Heliarch Containment 4-A: done"
+		not "Resistance: Questions: done"
+	source "Remote Blue"
+	destination "Station Cian"
+	on offer
+		conversation
+			`Hundreds of Heliarch agents pour out of the three Kimek transports, rushing in groups to separate parts of the city. They march in unison, weapons in hand. From the reactions of the locals, it's clear they were neither expecting such a thing, nor are they happy with the agents going about the city in that manner.`
+			`	When the last of the Heliarchs leave the transports, one of the Kimek captains inform you that you should now head to <destination> to collect your payment of <payment>, and to "avoid getting caught in any potential retaliation."`
+				accept
+	on complete
+		payment 495000
+		conversation
+			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortup. "For my absence, forgive me you must, Captain. Hit Fourth Shadow again, criminals have, so organizing some ships from within the Ring of Power, I am."`
+			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but trouble you with that, I will not. Very useful to us your support has been, Captain."`
```

Comment:
```suggestion
			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the Remote Blue operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Very useful to us your support has been, Captain."`
```